contracts/bitcoin_price_feed/BtcUsdPriceFeed.sol

```diff
@@ -49,6 +49,11 @@ contract BtcUsdPriceFeed is UsingWitnet, IERC2362 {
   function requestUpdate(uint256 _witnetRequestReward, uint256 _witnetResultReward, uint256 _witnetBlockReward) public payable {
     require(!pending, "An update is already pending. Complete it first before requesting another update.");
 
+    // Check whether we are covering gas prices
+    (uint256 minInclusionReward, uint256 minResultReward, uint256 minBlockReward) = witnetEstimateGasCost(tx.gasprice);
```

Comment:
Maybe we could change the naming of `minInclusionReward` to `minRequestReward`? so that goes with `_witnetRequestReward`, as the other 2 inputs have the same name

Reply:
maybe I would do the opposite, _witnetRequestReward to _witnetInclusionReward, as that is how the WRB is done